gallery/src/styles/NxDivider/NxDividerHorizontalExample.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { NxP, NxDivider, NxCode, NxButton, NxButtonBar } from '@sonatype/react-shared-components';
+import React from 'react';
+
+export default function NxDividerHorizontalExample() {
+  return (
+    <>
+      <NxP>This is a simple paragraph</NxP>
+      <NxDivider/>
+      <NxP>
+        This example simply demonstrates how to use the <NxCode>NxDivider</NxCode> to render
+        simple horizontal dividers.
+      </NxP>
+      <NxDivider />
```

Comment:
We demo styles using plain HTML elements and not convenience components.  In fact for especially simple setups such as this one, we typically write the demo in actual HTML rather than in JavaScript with react.  See most of the tile demos for example.

Reply:
> We demo styles using plain HTML elements and not convenience components. In fact for especially simple setups such as this one, we typically write the demo in actual HTML rather than in JavaScript with react. See most of the tile demos for example.

The example has been changed to in HTML rather than JS/React.
eb82df227306a3b7f7e0b0b35f58a3a7c488ef33